integration/apps/hpl_cpu/0002-Changed-LAinc-lib-to-the-correct-paths-and-openmp-fl.patch

```diff
@@ -0,0 +1,71 @@
+From 28146d103eaa9b2d03dd21c971c2fcf10cdda702 Mon Sep 17 00:00:00 2001
+From: Fuat Keceli <fuat.keceli@intel.com>
+Date: Fri, 9 Oct 2020 08:54:30 -0700
+Subject: [PATCH 2/3] Changed LAinc/lib to the correct paths and -openmp flag
+ to -qopenmp
+
+#  Copyright (c) 2015, 2016, 2017, 2018, 2019, 2020, Intel Corporation
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions
+#  are met:
+#
+#      * Redistributions of source code must retain the above copyright
+#        notice, this list of conditions and the following disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above copyright
+#        notice, this list of conditions and the following disclaimer in
+#        the documentation and/or other materials provided with the
+#        distribution.
+#
+#      * Neither the name of Intel Corporation nor the names of its
+#        contributors may be used to endorse or promote products derived
+#        from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+#  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+#  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+#  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+#  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+#  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+#  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+#  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY LOG OF THE USE
+#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+
+Signed-off-by: Fuat Keceli <fuat.keceli@intel.com>
+---
+ Make.Linux_Intel64 | 6 +++---
+ 1 file changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/Make.Linux_Intel64 b/Make.Linux_Intel64
+index 47661c2..205a459 100644
+--- a/Make.Linux_Intel64
++++ b/Make.Linux_Intel64
+@@ -94,10 +94,10 @@ HPLlib       = $(LIBdir)/libhpl.a
+ #
+ LAdir        = $(MKLROOT)
+ ifndef  LAinc
+-LAinc        = $(LAdir)/mkl/include
++LAinc        = $(LAdir)/include
+ endif
+ ifndef  LAlib
+-LAlib        = -L$(LAdir)/mkl/lib/intel64 \
++LAlib        = -L$(LAdir)/lib/intel64 \
+                -Wl,--start-group \
+                $(LAdir)/lib/intel64/libmkl_intel_lp64.a \
+                $(LAdir)/lib/intel64/libmkl_intel_thread.a \
+@@ -177,7 +177,7 @@ HPL_DEFS     = $(F2CDEFS) $(HPL_OPTS) $(HPL_INCLUDES)
+ #
+ CC       = mpiicc
```

Comment:
The compilers should all be updated to reference the build environment variables:
https://github.com/geopm/geopm/blob/dev/integration/config/build_env.sh#L54-61

Here this means:
```CC       = ${MPICC}```

Not sure if there are other compilers being used (Fortran?) but they should also be updated.

Reply:
Done.